services/src/main/java/io/scalecube/services/Microservices.java

```diff
@@ -309,4 +334,9 @@ public ProxyContext router(Class<? extends Router> router) {
     return servicesTags;
   }
 
+
+  public Transport transport() {
```

Comment:
Actually I think even further to add into Microservices config boolean property `reuseClusterTransport` so based either this property is set or not it will create new transport or reuse one from cluster.